src/lib/drishti/CMakeLists.txt

```diff
@@ -343,16 +343,18 @@ endforeach()
 ###### PUBLIC API ##########
 ############################
 
-foreach(library ${DRISHTI_SDK_3RDPARTY_LIBS})
-  get_target_property(_TARGET_TYPE "${library}" TYPE)
-  if(NOT _TARGET_TYPE STREQUAL "INTERFACE_LIBRARY")
-    install(FILES $<TARGET_FILE:${library}> DESTINATION 3rdparty)
-  endif()
-endforeach()
+if(DRISHTI_INSTALL_3RDPARTY_LIBS)
```

Comment:
I think it is better to remove the `DRISHTI_INSTALL_3RDPARTY_LIBS` option at this point.  This has been replaced (w/ a somewhat ad-hoc) method in the bash installation scripts here:

https://github.com/elucideye/drishti/blob/master/bin/make-release-utils.sh#L51-L91

This has some hand tuned logic to pull out the most common static libs from the hunter cache (along with the header files).  Obviously it is much cleaner to just use hunter if possible, and 3rdparty dependencies can be accessed from the hunter cache as needed.  I know you have thoughts about this as summarized here https://github.com/ruslo/hunter/issues/274